### DISCUSSION TITLE
How To Get Sponsors?

### DISCUSSION THREAD

**Discussion by 2030481** [community/community#63384] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How Can I get Sponsors on My GitHub Profile?
if you have any answer then please help.

**Comment by 2030481** [comment, marked as answer]
How i get more and more sponsors.

**Comment by workwithhim** [comment]
If you want to get the public sponsor badge on github profile, you can check the following link:
https://github.com/sponsors/caioagiani
